server/plugin/api.go

```diff
@@ -408,6 +408,9 @@ func (p *Plugin) completeConnectUserToGitHub(c *Context, w http.ResponseWriter,
 		return
 	}
 
+	// track the successful connection
+	p.tracker.TrackUserEvent("account_connected", c.UserID, nil)
```

Comment:
cc @m1lt0n

Reply:
1/5 on the name. the reasons why I chose that name:
- I wanted to make a difference from the wizard ones
- it's past (connected vs connect) because it already happened. we don't track intention but result here.